RepetitionAndConcatenation.py

```diff
@@ -0,0 +1,21 @@
+first_parameter = input("Enter first number: ")
+second_parameter = input("Enter the second number: ")
+
+
+def rep_cat(x, y):
+    if x.isdigit() and y.isdigit():
```

Comment:
The function _rep_cap_ is called only if x, y are digits; so this **if** is not mandatory.

Reply:
Removed it.